cmd/caib/main.go

```
@@ -964,16 +980,17 @@ func runShow(cmd *cobra.Command, args []string) {
+	fmt.Printf("  Export Format:       %s\n", build.Spec.ExportFormat)
+	fmt.Printf("  Mode:                %s\n", build.Spec.Mode)
 	fmt.Printf("  Manifest ConfigMap:      %s\n", build.Spec.ManifestConfigMap)
-	fmt.Printf("  OSBuild Image:      %s\n", build.Spec.AutomativeOSBuildImage)
-	fmt.Printf("  Storage Class:      %s\n", build.Spec.StorageClass)
-	fmt.Printf("  Serve Artifact:     %v\n", build.Spec.ServeArtifact)
-	fmt.Printf("  Serve Expiry Hours: %d\n", build.Spec.ServeExpiryHours)
+	fmt.Printf("  OSBuild Image:       %s\n", build.Spec.AutomativeOSBuildImage)
+	fmt.Printf("  Storage Class:       %s\n", build.Spec.StorageClass)
+	fmt.Printf("  Serve Artifact:      %v\n", build.Spec.ServeArtifact)
+	fmt.Printf("  Serve Expiry Hours:  %d\n", build.Spec.ServeExpiryHours)
+	fmt.Printf("  Server Route Expose: %s\n", build.Spec.ArtifactsRoute)
```

golangci-lint (1.64.8)

printf: fmt.Printf format %s has arg build.Spec.ArtifactsRoute of wrong type bool

(govet)